provider/resources.go

```diff
@@ -48,7 +48,10 @@ var metadata []byte
 func Provider() tfbridge.ProviderInfo {
 	// Instantiate the Terraform provider
 	p := pfbridge.MuxShimWithPF(context.Background(),
-		shimv2.NewProvider(provShim.SDKProvider()),
+		shimv2.NewProvider(
+			provShim.SDKProvider(),
+			shimv2.WithPlanResourceChange(func(_ string) bool { return true }),
```

Comment:
Nit: We don't need to have the `_` if every argument is ignored.
```suggestion
			shimv2.WithPlanResourceChange(func(string) bool { return true }),
```